src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+                "incidentId": "a86493e4-2d25-4023-a9e5-975bfaae19a8",
+                "policyIds": "195617",
+                "priority": "HIGH",
+                "state": "CREATED",
+                "title": "[\"PageView query result is > 1.3 for 2 minutes on 'RB PV '\"]",
+                "updatedAt": 1651058532782
+              },
+              {
+                "accountIds": "1",
+                "conditionFamilyId": "1284850",
+                "description": [
+                  "Policy: 'Itzik first policy'. Condition: 'error logs in nrb'"
+                ],
+                "incidentId": "5de80214-6444-4e0c-aaee-fb1c42fb2f23",
+                "policyIds": "707052",
+                "priority": "CRITICAL",
+                "state": "CREATED",
+                "title": "[\"notification-request-builder query result is < 10.0 for 1 minutes on 'error logs in nrb'\"]",
+                "updatedAt": 1651058523473
+              },
+              {
+                "description": [
+                  "Anomaly: Error rate"
+                ],
+                "incidentId": "df4ba4f3-9463-4389-873f-b13efb67dca5",
+                "priority": "MEDIUM",
+                "state": "CREATED",
+                "title": "[\"Anomaly: Error rate\"]",
+                "updatedAt": 1651058478828
+              },
+              {
+                "accountIds": "1",
+                "conditionFamilyId": "685305",
+                "description": [
+                  "Policy: 'Roi nov Test'. Condition: 'RB PV '"
+                ],
+                "incidentId": "0063f153-9152-476d-b435-54cc38ce0245",
+                "policyIds": "195617",
+                "priority": "CRITICAL",
+                "state": "CREATED",
+                "title": "[\"PageView query result is > 1.3 on 'RB PV '\"]",
+                "updatedAt": 1651058472738
+              },
+              {
+                "description": [
+                  "Anomaly: It was different from normal"
+                ],
+                "incidentId": "2cb820d1-a070-4c27-860b-fb862ecd42a7",
+                "priority": "MEDIUM",
+                "state": "CREATED",
+                "title": "[\"Anomaly: It was different from normal\"]",
+                "updatedAt": 1651058449078
+              },
+              {
+                "accountIds": "1",
+                "conditionFamilyId": "751197",
+                "description": [
+                  "Policy: 'slide_Test'. Condition: 'SumOf'"
+                ],
+                "incidentId": "07104b47-0263-4529-b27d-6aff95eecd87",
+                "policyIds": "199127",
+                "priority": "CRITICAL",
+                "state": "CREATED",
+                "title": "[\"TransactionError query result is > 3.0 on 'SumOf'\"]",
+                "updatedAt": 1651058442361
+              },
+
+```
+
+  </Collapser>
+  <Collapser
+     className="freq-link"
+     id="filterincidents"
+     title="Filter incidents"
+   >
+
+**Cursor pagination**
```

Comment:
for headers inside a collapser I tend to use H3s to start with, because h2s seem too heavy for inside a collapser and seem more prominent than the collapser text.